analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -850,6 +880,69 @@ async Task WithAsyncAwait()
     }
 }
 
+class Boxing
+{
+    void EmptyNullable()
+    {
+        int? nullable = null;
+        object implicitBoxed = nullable;
+        _ = (int)implicitBoxed;             // Compliant, true null-dereference -> no nullable value access
+        var explicitBoxed = (object)nullable;
+        _ = (int)explicitBoxed;             // Compliant
+        _ = (int)(object)(int?)(null);      // Compliant
+        _ = (int)(object)(null as int?);    // Compliant
+    }
+
+    void NullableRoundTrip()
+    {
+        int? nullable = null;
+        object implicitBoxed = nullable;
+        int? unboxedNullable = (int?)implicitBoxed;
+        _ = unboxedNullable.Value;          // Noncompliant, empty
+    }
```

Comment:
The test case where `nullable` is a parameter is missing.

Reply:
Defined [here](https://github.com/SonarSource/sonar-dotnet/pull/7006/files#diff-9e6c038bf5ad1e53d1b547b8870231cc2887c9c53296f73e3eb811f51613bdedR912)